### ISSUE TITLE
Sherlock is not working (for me at least)

### ISSUE THREAD

**Issue by RobinRMC** [fsociety-team/fsociety#32]
**Tool: Sherlock**<br>

Description: It doesn't execute the command because of an error.<br>
<br>

**Error: `sherlock: error: unrecognized arguments: -r`**<br>

Log:<br>

```bash
fsociety ~/information_gathering/# sherlock

Enter one or more usernames: Demo
usage: sherlock [-h] [--version] [--verbose] [--folderoutput FOLDEROUTPUT]
                [--output OUTPUT] [--tor] [--unique-tor] [--csv]
                [--site SITE_NAME] [--proxy PROXY_URL] [--json JSON_FILE]
                [--timeout TIMEOUT] [--print-all] [--print-found]
                [--no-color] [--browse] [--local]
                USERNAMES [USERNAMES ...]
sherlock: error: unrecognized arguments: -r
sherlock returned a non-zero exit code
```
<br>

**Fsociety information**<br>

```bash
# Config File

version = 3.2.5
agreement = true
ssh_clone = false
os = linux
host_file = hosts.txt
usernames_file = usernames.txt

# Python Version
3.9.2

# Platform
Linux-5.10.0-kali7-amd64-x86_64-with-glibc2.31
```

**Comment by thehappydinoa** [comment]
Hi there it was fixed in #29 but I need to cut a new release.

**Comment by thehappydinoa** [comment]
I just pushed v3.2.6. To upgrade run `pip install --upgrade fsociety`
